main/server/validations/common.js

```diff
@@ -0,0 +1,85 @@
+const t = require('io-ts')
```

Comment:
Lots of helpers to ensure valid data, can be seen as complex (and it is :disappointed: ) but then we can only use them.